circle-mlir/circle-mlir/lib/dialect/src/CircleDialect.cpp

```diff
@@ -426,7 +426,8 @@ Attribute ConstBytesAttr::parse(AsmParser &parser, Type type)
     return nullptr;
   }
 
-  std::string bytes_data = absl::HexStringToBytes(data.substr(2));
+  std::string bytes_data;
+  absl::HexStringToBytes(data.substr(2), &bytes_data);
```

Comment:
> function returns false and leaves `bytes` in an unspecified state

from the comment of this method, maybe ?
```suggestion
  if (not absl::HexStringToBytes(data.substr(2), &bytes_data))
  {
    return nullptr;
  }
```